app/controllers/albums.js

```diff
@@ -0,0 +1,14 @@
+const User = require('../models').user,
+  logger = require('../logger'),
+  fetch = require('node-fetch'),
+  sessionsManager = require('../services/sessionsManager'),
+  errors = require('../errors');
+
+exports.listAlbums = (req, res, next) => {
+  return fetch('https://jsonplaceholder.typicode.com/albums')
```

Comment:
Also this case it's super simple and it will sound unnecessary but its a good practice to try delegate responsibilities between different entities and not all in the controller. Here is a doc to read about this topic: https://docs.google.com/document/d/1ufn2NBFJkGyDmfDQCSWjBHrAQku0ZfZemc8aDV97qk8/edit?usp=sharing